src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -73,357 +56,214 @@ public AnnotationConfiguration LoadConfig()
                         {
-                            page.Stream.Close();
+                            page.SetAnnotations(AnnotationMapper.instance.MapForPage(annotations, i+1, info.PagesInfo[i]));
                         }
+
+                        if (pagesContent.Count > 0)
+                        {
+                            page.SetData(pagesContent[i]);
+                        }
+                        description.pages.Add(page);
                     }
                 }
-                string encodedImage = Convert.ToBase64String(bytes);
-                loadedPage.SetData(encodedImage);
-                DocumentInfoContainer documentDescription = AnnotationImageHandler.GetDocumentInfo(documentPath, password);
-                loadedPage.height = documentDescription.Pages[pageNumber - 1].Height;
-                loadedPage.width = documentDescription.Pages[pageNumber - 1].Width;                
-                // return loaded page object
-                return Request.CreateResponse(HttpStatusCode.OK, loadedPage);
-            }
-            catch (System.Exception ex)
-            {
-                // set exception message
-                return Request.CreateResponse(HttpStatusCode.InternalServerError, new Resources().GenerateException(ex, password));
             }
+
+            description.guid = documentGuid;
+            // return document description
+            return description;
         }
 
         /// <summary>
-        /// Download document
+        /// Get all annotations from the document
         /// </summary>
-        /// <param name="path">string</param>
-        /// <param name="annotated">bool</param>
-        /// <returns></returns>
-        [HttpGet]
-        [Route("downloadDocument")]
-        public HttpResponseMessage DownloadDocument(string path)
+        /// <param name="documentGuid">string</param>
+        /// <param name="documentType">string</param>
+        /// <returns>AnnotationInfo[]</returns>
+        private AnnotationBase[] GetAnnotations(string documentGuid, string documentType, string password)
         {
-            // prepare response message
-            HttpResponseMessage response = new HttpResponseMessage(HttpStatusCode.OK);          
-           
-            // add file into the response
-            if (File.Exists(path))
+            try
             {
-                
-                var fileStream = GetCleanDocumentStream(path);
-                response.Content = new StreamContent(fileStream);
-                response.Content.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
-                response.Content.Headers.ContentDisposition = new ContentDispositionHeaderValue("attachment")
+                FileStream documentStream = new FileStream(documentGuid, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+                DocumentType docType = DocumentTypesConverter.GetDocumentType(documentType);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'docType'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)